features/openshift.feature

```diff
@@ -47,9 +48,10 @@ Feature: Command output from various OpenShift related commands
     """
     OPENSHIFT_URL=https://<ip>:8443
     OPENSHIFT_WEB_CONSOLE=https://<ip>:8443/console
+    DOCKER_REGISTRY=hub.openshift.rhel-cdk.<ip>.xip.io
```

Comment:
+1

We should probably add another test here where we call _env_ without explicit _openshift_ and verify that all variables are printed.